microbench/pom.xml

```diff
@@ -39,8 +39,25 @@
       <artifactId>alluxio-core-server-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-master</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-tests</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
 
     <!-- external dependencies -->
+    <dependency>
+      <groupId>site.ycsb</groupId>
```

Comment:
what license is this governed by?

Reply:
i believe apache, but why do we need ycsb for this partial listing PR?

Reply:
answering my own question, it gets ycsb depedency from the benchmarking merges

Reply:
Apache 2.0 https://github.com/brianfrankcooper/YCSB

This is from this PR which this one depends on.
https://github.com/Alluxio/alluxio/pull/15738/files

Reply:
Yea I merged with the benchmark PR because I wanted to make sure it didn't impact performance too much.